cartridge-cli.lua

```diff
@@ -3424,15 +3497,54 @@ function cmd_pack.callback(args)
         )
     end
 
+    local sdk_path
     if app_state.tarantool_is_enterprise and app_state.build_in_docker then
-        if app_state.download_token == nil or app_state.download_token == '' then
-            die(
-                'Tarantool download token is required to pack enterprise Tarantool app in docker. ' ..
-                'Please, specify it using --download_token option or TARANTOOL_DOWNLOAD_TOKEN env variable'
-            )
+        -- check that passed one option for SDK
+        local sdk_params_are_right = check_that_only_one_is_true({
+            args.sdk_local,
```

Comment:
Consider a way to validate what you will download or copy.

You should not to copy MacOs binary inside CentOS
You should gracefully die if download url leads to something else except Tarantool EE.

Reply:
I'm agree with you about dying for bad download URL, but I'm not sure that we should check binaries.

Option `--sdk-local` introduced to prevent using some default scenarios (like copying local SDK). The main idea was to shift responsibility for choosing right bundle to user.

I'm thinking about removing `--sdk-url` option in favor to using only `--sdk-path`. I think that it would be better if user downloads and unpacks SDK by himself.

Reply:
> we should check binaries

Should be enough to call `tarantool --version` and check that it doesn't contain "Darwin" and "FreeBSD" (not sure about this).

Reply:
> I'm thinking about removing --sdk-url option in favor to using only --sdk-path. I think that it would be better if user downloads and unpacks SDK by himself.

Sounds reasonable